plugins/providers/bigquery/activity.go

```diff
@@ -8,10 +8,8 @@ import (
 	"strings"
 	"time"
 
-	"cloud.google.com/go/logging"
-	"cloud.google.com/go/logging/logadmin"
 	"github.com/goto/guardian/domain"
-	"google.golang.org/api/iterator"
+	"google.golang.org/api/logging/v2"
```

Comment:
Any specific reason we are changing the library being used?

Reply:
we now use getLoggingBucket API which was not supported in `cloud.google.com/go/logging`, so rather than using two different clients I replace the other calls to use this `logging/v2` instead.